python/orca/src/bigdl/orca/learn/pytorch/pytorch_pyspark_worker.py

```diff
@@ -29,7 +29,6 @@
 # limitations under the License.
 
 
-from subprocess import call
 from pyspark import BarrierTaskContext
 from bigdl.orca.learn.pytorch.torch_runner import TorchRunner
 import torch.distributed as dist
```

Comment:
logging and save pkl are not needed?

Reply:
`logging` is used in Line 47. `save_pkl` is duplicated